tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js

```diff
@@ -925,1366 +813,852 @@ const cldrLoad = (function () {
-                true
-              ); /* show the box the first time */
-            } else if (!json.canModify) {
-              cldrSurvey.showInPop2(
-                cldrText.get("readonlyGuidance"),
-                null,
-                null,
-                null,
-                true
-              ); /* show the box the first time */
-            } else {
-              cldrSurvey.showInPop2(
-                cldrText.get("dataPageInitialGuidance"),
-                null,
-                null,
-                null,
-                true
-              ); /* show the box the first time */
-            }
-            if (!cldrSurvey.isInputBusy()) {
-              cldrSurvey.showLoader(cldrText.get("loading3"));
-              cldrTable.insertRows(
-                theDiv,
-                json.pageId,
-                cldrStatus.getSessionId(),
-                json
-              ); // pageid is the xpath..
-              cldrSurvey.updateCoverage(flipper.get(pages.data)); // make sure cov is set right before we show.
-              flipper.flipTo(pages.data); // TODO now? or later?
-              showCurrentId(); // already calls scroll
-              cldrSurvey.refreshCounterVetting();
-              $("#nav-page-footer").show(); // make bottom "Prev/Next" buttons visible after building table
-            }
-          }
-        });
+        loadAllRows(itemLoadInfo, theDiv);
       }
-    } else if (cldrStatus.getCurrentSpecial() == "oldvotes") {
-      const curLocale = cldrStatus.getCurrentLocale();
-      var url =
-        cldrStatus.getContextPath() +
-        "/SurveyAjax?what=oldvotes&_=" +
-        curLocale +
-        "&s=" +
-        cldrStatus.getSessionId() +
-        "&" +
-        cldrSurvey.cacheKill();
-      myLoad(url, "(loading oldvotes " + curLocale + ")", function (json) {
-        isLoading = false;
-        cldrSurvey.showLoader(cldrText.get("loading2"));
-        if (!verifyJson(json, "oldvotes")) {
-          return;
-        } else {
-          cldrSurvey.showLoader("loading..");
-          if (json.dataLoadTime) {
-            cldrSurvey.updateIf("dynload", json.dataLoadTime);
-          }
-
-          var theDiv = flipper.flipToEmpty(pages.other); // clean slate, and proceed..
+    } else if (curSpecial === "none") {
+      cldrSurvey.hideLoader();
+      isLoading = false;
+      window.location = cldrStatus.getSurvUrl(); // redirect home
+    } else {
+      const special = getSpecial(curSpecial);
+      if (special) {
+        special.load();
+      } else {
+        console.log("No special js found for " + curSpecial);
+      }
+    }
+  }
 
-          cldrSurvey.removeAllChildNodes(theDiv);
+  function getSpecial(str) {
+    const map = {
+      about: cldrAbout,
+      createAndLogin: cldrCreateLogin,
+      forum: cldrForum,
+      locales: cldrLocales,
+      mail: cldrMail,
+      oldvotes: cldrOldVotes,
+      r_compact: cldrReportNumbers,
+      r_datetime: cldrReportDate,
```

Comment:
mismatch between the object and file names: "cldrReportDate" vs. "cldrReportDates" -- harmless but I'll add the "s" to the object name for consistency